src/erlfmt_format.erl

```diff
@@ -491,18 +488,13 @@ is_tag(Expr) ->
 is_container(Expr) ->
     lists:member(element(1, Expr), [tuple, bin, list, map, record, call, macro_call, lc, bc]).
 
-surround_container(line, Left, Doc, Right) ->
-    surround(Left, <<"">>, concat(force_breaks(), Doc), <<"">>, Right);
-surround_container(break, Left, Doc, Right) ->
-    surround(Left, <<"">>, Doc, <<"">>, Right);
-surround_container(group_line, Left, Doc, Right) ->
-    surround(Left, <<"">>, concat(force_breaks(), group(Doc)), <<"">>, Right);
-surround_container(group_break, Left, Doc, Right) ->
-    surround(Left, <<"">>, group(Doc), <<"">>, Right);
 surround_container(no_break, Left, Doc, Right) ->
     concat(Left, Doc, Right);
 surround_container(flex_break, Left, Doc, Right) ->
-    group(concat(nest(concat(Left, Doc), ?INDENT, break), Right)).
+    group(concat(nest(concat(Left, Doc), ?INDENT, break), Right));
+surround_container(#break{inner = ForceInner, outer = ForceOuter}, Left, Doc, Right) ->
+    InnerDoc = group(concat(maybe_force_breaks(ForceInner), Doc)),
```

Comment:
So Doc is now always grouped, does this still cover the `line` case, which was not grouped?

Reply:
Yes, using `force_breaks()` turns `break()` into `line()` so that still works correctly and removes one case